src/demo/pages/ButtonPage/examples/Button.Basic.Example.scss

```diff
@@ -1,3 +1,8 @@
 .ms-BasicButtonsExample .ms-Button {
   margin: 10px 0;
 }
+
+.ms-Icon {
+  margin-right: 5px;
+  margin-left: -10px;
```

Comment:
use the $include margin-left(...) macros for RTL sensitive rules. (just search around for $include examples, you see)